README.md

```diff
@@ -92,4 +92,12 @@ export MOCK_GS1_SERVICE_API_KEY=test456 # Mock GS1 service API key
 # Run seeding scripts
 ./seeding/idr-data.sh
 ./seeding/mock-gs1-data.sh
+```
+
+## End-to-end testing
+```bash
+# Run end-to-end testing scripts
+docker-compose up -d
+yarn start:e2e-test
+yarn cypress run
 ```
```

Comment:
I've tried these instructions, but they didn’t work for me.
Do we need anything else to make it work?

<img width="1750" alt="image" src="https://github.com/user-attachments/assets/ecbf9664-f231-441a-8b8f-2e4c9beeddd6" />

Reply:
Ah, I forgot to run `yarn install` again. My bad.

Reply:
But I encountered another error, as shown below.
Do I need anything else to make it work?

---

Terminal Console Output:

<img width="1750" alt="image" src="https://github.com/user-attachments/assets/9d049488-389d-4b19-8544-50433d3093e5" />

Cypress generated report:

![Facility Links Test -- should visit the homepage, navigate to Orchard Facility, handle API calls, and show success message (failed)](https://github.com/user-attachments/assets/f673a2ce-3a92-4159-8a95-f4e4248cec01)